app/Resources/views/home/index.html.twig

```diff
@@ -154,15 +154,15 @@
             <form name="home_donation" method="post">
                 <div class="donate__amounts">
                     <input type="radio" id="home_donation_amount_10" name="home_donation[amount]" value="€10" />
-                    <label for="home_donation_amount_10" class="amount">€10</label>
+                    <label for="home_donation_amount_10" class="amount">10€</label>
                     <input type="radio" id="home_donation_amount_20" name="home_donation[amount]" value="€20" />
-                    <label for="home_donation_amount_20" class="amount">€20</label>
+                    <label for="home_donation_amount_20" class="amount">20€</label>
                     <input type="radio" id="home_donation_amount_30" name="home_donation[amount]" value="€30" />
-                    <label for="home_donation_amount_30" class="amount">€30</label>
+                    <label for="home_donation_amount_30" class="amount">30€</label>
                     <input type="radio" id="home_donation_amount_100" name="home_donation[amount]" value="€100" />
-                    <label for="home_donation_amount_100" class="amount">€100</label>
+                    <label for="home_donation_amount_100" class="amount">100€</label>
                     <input type="radio" id="home_donation_amount_150" name="home_donation[amount]" value="€150" />
-                    <label for="home_donation_amount_150" class="amount">€150</label>
+                    <label for="home_donation_amount_150" class="amount">150€</label>
```

Comment:
I did this € inversion in #44, do you think you can revert here to avoid a conflict later :) ?